style_editor.html

```diff
@@ -9166,9 +9169,9 @@
     overlay.classList.remove("show");
   }
 
-  function DontShowAgain() {
-    localStorage.setItem(popupIdentifier, "false");
-  console.log("Saving " + popupIdentifier + " false");
+  function DontShowAgain(checkboxState) {
+    localStorage.setItem(popupIdentifier, checkboxState ? "false" : "true");
```

Comment:
foo ? "false" : "true" is werd

why not just do: checkboxState = !checkBoxState at the beginning of the function so that the conditions make sense? (And maybe we can just do:
localStorage.setItem(popupIdentifier, checboxState); ?)

Reply:
Better readability, sure. Fixed.